src/gtp_socket.erl

```diff
@@ -379,6 +399,10 @@ make_request(ArrivalTS, IP, Port,
 family({_,_,_,_}) -> inet;
 family({_,_,_,_,_,_,_,_}) -> inet6.
 
+% this returns family with the version number in the end
+family_v({_,_,_,_}) -> inet4;
+family_v({_,_,_,_,_,_,_,_}) -> inet6.
```

Comment:
I think this is not needed. The only valid use case is currently gen_socket:bind. However, there are enough differences between IPv4 and IPv6 sockets  (and there might be even more) that a separate bind helper makes sense (see gtp_socket:bind_gtp_socket/3).